### PR TITLE
Add unit test coverage for fhir router graphql

### PR DESCRIPTION
**Pull request by tonylee80**
### Summary
🧪🔗🚀

Added a test case for the GraphQL connection API without the count field. This tests the performance and correctness of the custom FHIR pagination and filtering extension in `packages/fhir-router/src/graphql.test.ts`.

> _`connection` test_
> _without the costly `count` field_
> _autumn leaves fall fast_

### Walkthrough
* Add a test case for the GraphQL connection API without the count field ([link](https://github.com/medplum/medplum/pull/1946/files?diff=unified&w=0#diff-a0896bc706df6ce472112e4f8181a84141bdeb5fab7ec9fa85896ef053be94bbR979-R1011)). This test verifies that the connection API can handle queries that do not request the total count of matching resources, which can be expensive to compute. The connection API is a custom extension of the FHIR standard that allows pagination and filtering of FHIR resources using GraphQL syntax. The test file is `packages/fhir-router/src/graphql.test.ts`.

### CODE REVIEW

File: packages/fhir-router/src/graphql.ts [hunk 1]
```diff
@@ -157,16 +157,13 @@ export async function graphqlHandler(req: FhirRequest, repo: FhirRepository): Pr
 
   const dataLoader = new DataLoader<Reference, Resource>((keys) => repo.readReferences(keys));
 
-  let result: any = introspection && introspectionResults.get(query);
```
Comment (tonylee80) [inline packages/fhir-router/src/graphql.ts:160]: due to added guard above at https://github.com/medplum/medplum/pull/1946/files#diff-edf7ac6d441b8b49c1649bdc9c7da39a333b108c8494d4324eaaf5d19a29eff5L154, `introspection` will always be false here.

File: packages/fhir-router/src/graphql.test.ts [hunk 2]
```diff
@@ -976,4 +976,37 @@ describe('GraphQL', () => {
       edges: [{ resource: { name: [{ given: ['Alice'] }] } }],
     });
   });
+
+  test('Connection API without count field', async () => {
```
Comment (tonylee80) [inline packages/fhir-router/src/graphql.test.ts:980]: adds coverage for https://sonarcloud.io/code?id=medplum_medplum&pullRequest=1934&selected=medplum_medplum%3Apackages%2Ffhir-router%2Fsrc%2Fgraphql.ts&line=590

File: packages/fhir-router/src/graphql.ts (hunk 1 above)
Comment (tonylee80) [reply packages/fhir-router/src/graphql.ts]: added the code back, just noticed #1937 where introspection support would be managed through a server config.

**Review by codyebberson — APPROVED** [review]
🎉

**Pull request merged**